awx/api/serializers.py

```diff
@@ -1396,7 +1396,6 @@ def validate(self, attrs):
 
 class ExecutionEnvironmentSerializer(BaseSerializer):
     show_capabilities = ['edit', 'delete', 'copy']
-    managed_by_tower = serializers.ReadOnlyField()
```

Comment:
we should keep this, imo. removing it will cause headaches for us later down the line and cause the field to be too flexible